qa/workunits/rbd/import_export.sh

```diff
@@ -79,10 +79,18 @@ if rbd help export | grep -q export-format; then
     dd if=/bin/dd of=${TMPDIR}/img bs=1k count=10 seek=100
     rbd import $RBD_CREATE_ARGS ${TMPDIR}/img testimg
     rbd snap create testimg@snap
+    rbd image-meta set testimg key1 value1
+    KEY=`rbd image-meta list testimg|grep key1`
+    VALUE=`rbd image-meta list testimg|grep value1`
```

Comment:
The $KEY and $VALUE seems refer to the same line. right? because the key1 and value1 should be outputted in the same line.